### PR TITLE
OPGOPS-1808 Changed jenkins call by token urls

### PR DESCRIPTION
**Pull request by carinadigital**
WIP for the moment as it may break current slaves.

### CODE REVIEW

**Comment by carinadigital** [comment]
The format of URL will work with our older jenkins install

File: jenkins-slave/docker/etc/my_init.d/999_register_jenkins_slave [hunk 1]
```diff
@@ -39,7 +39,10 @@ class DockerConfiguration:
         json_data = json.loads(json_string)[0]
         host_bindings = json_data['NetworkSettings']['Ports']
         self.host_ip_address = self._get_host_ip()
-        self.host_ssh_port = host_bindings['22/tcp'][0]['HostPort']
+        if '22/tcp' in host_bindings and host_bindings['22/tcp'] is not None and 'HostPort' in host_bindings['22/tcp'][0]:
```
Comment (brettminnie) [inline jenkins-slave/docker/etc/my_init.d/999_register_jenkins_slave]: not pep8 compliant you'll have to split the lines here

File: jenkins-slave/docker/etc/my_init.d/999_register_jenkins_slave [hunk 2]
```diff
@@ -116,6 +118,7 @@ class SelfRegistration:
 
     def register(self):
         request = urllib2.Request(self.jenkins_url)
+        print "Trying to connect to {}".format(self.jenkins_url.replace(os.environ['JENKINS_AUTH_TOKEN'], '[REDACTED]'))
```
Comment (brettminnie) [inline jenkins-slave/docker/etc/my_init.d/999_register_jenkins_slave]: pep8

**Review by brettminnie — CHANGES_REQUESTED** [review]
Looks good, just need to make the script pep8 compliant

**Review by brettminnie — APPROVED** [review]

**Pull request merged**